src/RoslynPad.Avalonia/DocumentViewFoldingManager.cs

```diff
@@ -0,0 +1,145 @@
+﻿using Avalonia.Threading;
+using AvaloniaEdit.Folding;
```

Comment:
Following the above, please add a bool property to RoslynCodeEditor that controls whether folding is enabled,

Reply:
Fixed